Makefile

```diff
@@ -262,6 +262,9 @@ cloud-images: ## Uploads cloud images (AMIs, etc.) to the cloud registry.
 
 # Code Quality
 
+api-descriptors: ## Generates API descriptors used to detect breaking API changes.
```

Comment:
That target should be called before or right after release

Reply:
I wonder if we can call it from the `hack/release.sh` script.

Reply:
tbh, I'm not sure. It might be better to update descriptors right after release. At least that's how I used that tool previously, but there were no breaking API changes in that project; Talos will have them before 1.0.

Reply:
it could be that we even run it after each API change, can't we? (if we proved that it's backwards compatible)

but probably each release makes more sense